app/models/concerns/smooch_messages.rb

```diff
@@ -344,7 +344,7 @@ def save_message(message_json, app_id, author = nil, request_type = 'default_req
         message['archived'] = (request_type == 'relevant_search_result_requests' ? self.default_archived_flag : CheckArchivedFlags::FlagCodes::UNCONFIRMED)
         annotated = self.create_project_media_from_message(message)
         if annotated != annotated_obj && annotated.is_a?(ProjectMedia)
-          Relationship.create!(relationship_type: Relationship.suggested_type, source: annotated_obj, target: annotated, user: BotUser.smooch_user)
+          Relationship.create(relationship_type: Relationship.suggested_type, source: annotated_obj, target: annotated, user: BotUser.smooch_user)
```

Comment:
I removed the `!` because we just want to ignore really if a suggestion already exists.